validation/policies/io/konveyor/forklift/vmware/uefi_boot.rego

```diff
@@ -8,7 +8,7 @@ concerns[flag] {
     has_uefi_boot
     flag := {
         "category": "Warning",
-        "label": "UEFI secure boot detected",
-        "assessment": "UEFI secure boot is enabled. NVRAM data is not copied during the migration and thus the VM might not boot on OpenShift Virtualization."
+        "label": "UEFI detected",
+        "assessment": "If UEFI secure boot is enabled, NVRAM data is not copied during the migration and thus the VM might not boot on OpenShift Virtualization."
```

Comment:
I would explicitly mention that the migtrated VM will have secure boot disabled.

Reply:
yeah, maybe something like:
```suggestion
        "assessment": "UEFI secure boot will be disabled on Openshift Vrtualization. If the VM was set with UEFI secure boot, manual steps within the guest would be needed in order to boot the guest operating system."
```